### PR TITLE
chore: update deps

### LINKED ISSUE #10066: closing laptop lid on deno run --watch causes "interrupted system call" error
interesting error that occurred when I shut my laptop lid and reopened it on a watched process

`deno run --unstable --watch file.ts`
```
Watcher Process finished! Restarting on file change...
thread '<unnamed>' panicked at 'poll failed: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/notify-5.0.0-pre.5/src/inotify.rs:134:47
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

the watcher appears to stop working after this error is shown, but the process does not exit

### PR DESCRIPTION
**Pull request by lucacasonato**
Update all the deps.

Closes #10066

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -731,20 +737,21 @@ dependencies = [
  "serde",
  "tokio",
  "tokio-rustls",
- "tokio-tungstenite",
+ "tokio-tungstenite 0.14.0",
```
Comment (bartlomieju) [inline Cargo.lock:740]: Looks like there are now 2 versions of `tokio-tungstenite`

File: Cargo.lock [hunk 2]
```diff
@@ -1523,8 +1527,8 @@ dependencies = [
  "httparse",
  "httpdate",
  "itoa",
- "pin-project 1.0.5",
- "socket2",
+ "pin-project",
+ "socket2 0.4.0",
```
Comment (bartlomieju) [inline Cargo.lock:1531]: Two versions of `socket2`?

File: Cargo.lock [hunk 3]
```diff
@@ -3515,9 +3502,26 @@ checksum = "e1a5f475f1b9d077ea1017ecbc60890fda8e54942d680ca0b1d2b47cfa2d861b"
 dependencies = [
  "futures-util",
  "log",
- "pin-project 1.0.5",
+ "pin-project",
+ "tokio",
+ "tungstenite 0.12.0",
```
Comment (bartlomieju) [inline Cargo.lock:3507]: Multiple versions of `tungstenite`?

File: Cargo.lock (hunk 1 above)
Comment (lucacasonato) [reply Cargo.lock]: Whoops, forgot to update test_util/Cargo.toml. We have so many `Cargo.toml` now

**Review by bartlomieju — APPROVED** [review]
LGTM

File: Cargo.lock (hunk 3 above)
Comment (lucacasonato) [reply Cargo.lock]: Resolved

File: Cargo.lock (hunk 2 above)
Comment (lucacasonato) [reply Cargo.lock:1531]: We can't do anything about this. `hyper` needs the new one, while `trust-dns-resolver` uses the old one.

**Pull request merged**